pythoncalculator/subtract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x, y)
```

Comment:
This line should have a colon at the end. Can you add it please?

Reply:
Sorted 😃